### PR TITLE
Initialize project with components

### PR DESCRIPTION
**Pull request by Rhaegar121**
In this PR, I have completed the following tasks.

1. Setup linters for react
2. Used correct gitflow
3. Added two routers: `/books` and `/categories`
4. Created `Navbar` component
5. Created `Book` component
6. Created `Form` component

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,24 +1,31 @@
...
+import Categories from './compoents/Categories';
+
+function Layout() {
+  return (
+    <>
+      <Navbar />
+      <Outlet />
+    </>
+  );
+}
```
Comment (mckent05) [inline src/App.js]: - Good job done so far, I suggest having this component in another js file, so as to make your code as modular as possible 👍 👍 
- I also advise using arrow functions in order to maintain `ES6` syntax as it is the more accepted syntax in use globally. Please fix this 👍 👍

File: src/App.js [hunk 2]
```diff
@@ -1,24 +1,31 @@
...
+      <Routes>
+        <Route path="/" element={<Layout />}>
+          <Route index element={<Book title="book1" author="Barry" />} />
+          <Route path="categories" element={<Categories />} />
+          <Route path="*" element={<h1>Not found</h1>} />
+        </Route>
+      </Routes>
+    </BrowserRouter>
   );
 }
```
Comment (mckent05) [inline src/App.js]: - I advise using arrow functions in order to maintain ES6 syntax as it is the more accepted syntax in use globally. Please fix this 👍 👍

File: src/compoents/Book.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+/* eslint-disable react/prop-types */
```
Comment (mckent05) [inline src/compoents/Book.js]: - Please remove all disabled linter checks, remember that linters help us write code that is clean and easy to maintain. Please fix this 🙏

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @Rhaegar121, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- You have setup your components correctly and your app renders properly 👏 👏 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 4]
```diff
@@ -29,42 +110,69 @@ Your app is ready to be deployed!
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-### Deployment
+<!-- LICENSE -->
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+## 📝 License <a name="license"></a>
 
-### `npm run build` fails to minify
+This project is [MIT](./LICENSE) licensed.
```
Comment (Qoosim) [inline README.md:176]: - [OPTIONAL] This link is not broken. Kindly follow this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to properly set it up for your project.

license

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @Rhaegar121, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct Git Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**